web/src/components/diff/FileDiffNode.tsx

```diff
@@ -76,13 +104,14 @@ export class FileDiffNode extends React.PureComponent<FileDiffNodeProps, State>
                                 <ChevronRightIcon className="icon-inline" />
                             )}
                         </button>
-                        <div className="file-diff-node__header-path-stat">
-                            <DiffStat
-                                added={node.stat.added}
-                                changed={node.stat.changed}
-                                deleted={node.stat.deleted}
-                                className="file-diff-node__header-stat"
-                            />
+                        <div className="file-diff-node__header-path-stat align-items-baseline">
+                            {stat}
+                            {!node.oldPath && (
+                                <span className="file-diff-node__header-path-info text-success mr-2">New file</span>
```

Comment:
I think a badge would look better, the monospace font for prose is a bit hard to read/unconventional
```suggestion
                                <span className="badge badge-success mr-2">New file</span>
```

Reply:
Maybe also use "added" instead of "new" since that is how git status/diff calls it
```suggestion
                                <span className="file-diff-node__header-path-info text-success mr-2">Added file</span>
```